.env.example

```
@@ -1,7 +1,22 @@
-NETBOX_URL=
-NETBOX_TOKEN=
+NETBOX_URL=https://netbox.example.org
+NETBOX_TOKEN=XXXXXXXXX-xxxxxxxxx
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NETBOX_TOKEN key should go before the NETBOX_URL key

(UnorderedKey)